aspnetcore/host-and-deploy/health-checks.md

```diff
@@ -0,0 +1,362 @@
+---
+title: Health checks in ASP.NET Core
+author: guardrex
+description: Learn how to set up health checks for ASP.NET Core infrastructure, such as apps and databases.
+monikerRange: '>= aspnetcore-2.2'
+ms.author: riande
+ms.custom: mvc
```

Comment:
mvc???

Reply:
:smile: I asked that, too ...... *mvc* == "Minimum Viable Content" (I think)

Reply:
I'm going to start using that 👍

Reply:
I shouldn't say this, but APEX does some *very interesting things*. :smile: lol